### PR TITLE
Tool-suite commander cleanup and update

### LINKED ISSUE #1064: Add infotainment Simulator-Reset-Button
To let the driver reset the scenario during the drive. 
Probably an additional button is needed, something with *maintenance* or *help* or *reset* style icon.
When such button is pressed, CMD_RESTART(BEAM_NG_SCENARIO) message is sent.

### PR DESCRIPTION
**Pull request by podhrmic**
TLDR; the purpose of this PR is to bring the demonstrator to a functional state *without* the hearbeat/system monitor feature.

Quick summary:
* make tool-suite single-threaded for the purpose of Cyberphys mode
* make tool-suite work on two separate computers with different targets (3xFreeRTOS, 3xDebian) at *the same time*
* Add reset button to the infotainment
* Massive code cleanup & performance improvements
* Test suite for the kiosk/scenarios

Fixes #1064 #1262 #1253

### CODE REVIEW

File: besspin/cyberPhys/infotainmentserver.py [hunk 1]
```diff
@@ -53,8 +53,7 @@ def restart (xTarget):
         # the kill_listeners.sh script
         #xTarget.runCommand("pkill infotainment_se",exitOnError=False,tee=appLog)
         xTarget.runCommand("/opt/kill_listeners.sh",exitOnError=False,tee=appLog)
-        xTarget.runCommand("systemctl stop infotainment-server.service", erroneousContents=["Failed to stop", "error code"], tee=appLog)
-        xTarget.runCommand("systemctl start infotainment-server.service", erroneousContents=["Failed to start", "error code"], tee=appLog)
+        xTarget.runCommand("systemctl restart infotainment-server.service", erroneousContents=["Failed", "error code"], tee=appLog)
```
Comment (dmzimmerman) [inline besspin/cyberPhys/infotainmentserver.py:56]: I realize we don't care much about FreeBSD, but shouldn't we change it to `restart` there too?

File: besspin/cyberPhys/otaserver.py [hunk 2]
```diff
@@ -53,8 +53,7 @@ def restart (xTarget):
     printAndLog(f"{xTarget.targetIdInfo}Restarting ota service!",tee=appLog)
     serviceTimeout = 120
     if isEqSetting('osImage','debian',targetId=xTarget.targetId):
-        xTarget.runCommand("systemctl stop ota.service", erroneousContents=["Failed to stop", "error code"],tee=appLog)
-        xTarget.runCommand("systemctl start ota.service", erroneousContents=["Failed to start", "error code"],tee=appLog)
+        xTarget.runCommand("systemctl restart ota.service", erroneousContents=["Failed", "error code"],tee=appLog)
```
Comment (dmzimmerman) [inline besspin/cyberPhys/otaserver.py:56]: (and here as well, for the `restart`)

File: besspin/cyberPhys/specs/infotainment.txt [hunk 3]
```diff
@@ -2,18 +2,19 @@
...
-Mux -> Mux: Update music/volume state from active server
-Mux -[#red]> ThinClient: Forward INFOTAINMENT_STATE
-ThinClient --> Driver: displays respose
+ThinClient -[#red]> Server: broadcast CAN_ID_BUTTON_PRESSED to port 5002
+Server -[#red]> Ignition: broadcast CAN_ID_INFOTAINMENT_STATE to port 5002
+Ignition -> Ignition: Receive only from the server on the active network
+Ignition -> Ignition: Update music/volume state
+Ignition -> Ignition: Poll simulator for position updates
+
+Ignition -[#green]> ThinClient: Send CAN_ID_CAR_X/Y/Z/X to port 5013
```
Comment (dmzimmerman) [inline besspin/cyberPhys/specs/infotainment.txt:16]: Only from the server on the active network.

File: besspin/cyberPhys/infotainmentserver.py (hunk 1 above)
Comment (podhrmic) [reply besspin/cyberPhys/infotainmentserver.py:56]: Fixed

File: besspin/cyberPhys/otaserver.py (hunk 2 above)
Comment (podhrmic) [reply besspin/cyberPhys/otaserver.py:56]: Fixed

File: besspin/cyberPhys/specs/infotainment.txt (hunk 3 above)
Comment (podhrmic) [reply besspin/cyberPhys/specs/infotainment.txt:16]: Updated

**Review by dmzimmerman — APPROVED** [review]
LGTM

**Comment by podhrmic** [comment]
Merging as is, as it addresses the issues mentioned.

**Pull request merged**